packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -52,6 +53,43 @@ async function withSFConnection(f: (conn: Connection) => void): Promise<void> {
   }
 }
 
+class ConnectionChangedListener {
```

Comment:
Is this PR only taking care of the connection handling ? I don't see the editor handling the event/message sent when switching connections (`onConnectionChange`).

Reply:
Yes. The editor code is here: https://github.com/forcedotcom/soql-tooling/pull/60

Reply:
Cool 👍